02-s3/lab-2.3.1/lab.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+export AWS_PAGER=
+
+# was already using mostly cfn, but this is a little better!
+
+aws_sl='aws-vault exec stelligent_labs -- aws'
+
+$aws_sl cloudformation deploy --template-file policy.cfn.yaml --stack-name su-lab-2-3-1-jdix --capabilities CAPABILITY_NAMED_IAM --region us-west-2
+$aws_sl s3 sync ./data s3://stelligent-u-josh-dix-labs/data/ --region us-west-2
+
+echo "making changes to data files"
+echo "changes" >> ./data/private.txt
+echo "changes" >> ./data/t1.txt
+echo "changes" >> ./data/t2.txt
+echo "changes" >> ./data/t3.txt
+echo "changes" >> ./data/t4.txt
```

Comment:
massive nitpick, no need to change, but using a for loop here is probably better; something like

```suggestion
for i in $(ls *.txt); do echo "changes" >> $i; done
```